pkgs/applications/networking/mailreaders/meli/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchgit
+, rustPlatform
+, pkgconfig
+, openssl
+, dbus
+, sqlite
+, file
+, gzip
+, notmuch
+, withNotmuch ? true
```

Comment:
Can you leave a comment about what "withNotmuch" changes? Is it using Notmuch as a mail backend?